spec/features/dams_object_spec.rb

```diff
@@ -352,3 +352,27 @@ class << self
     expect(page).to have_content('mads:Name External')
   end
 end
+
+describe "Object with internal-only notes" do
+  before(:all) do
+    notes = [{value: 'public note'}, {value: 'internal note', internalOnly: true}]
+    @unit = DamsUnit.create name: "Test Unit", description: "Test Description", code: "tu", uri: "http://example.com/"
+    @obj = DamsObject.create unitURI: @unit.pid, titleValue: "Note Test Object", copyright_attributes: [{status: 'Public domain'}], note_attributes: notes
+    solr_index @obj.pid
+  end
+  after(:all) do
+    #@obj.delete
```

Comment:
Is this intentionally commented out? If so, where is the cleanup for the notes created for these tests?